plugins/main/src/main/kotlin/app/aaps/plugins/main/general/overview/OverviewFragment.kt

```diff
@@ -1146,13 +1146,23 @@ class OverviewFragment : DaggerFragment(), View.OnClickListener, OnLongClickList
         if (variableSens != isfMgdl && variableSens != 0.0 && isfMgdl != null) {
             var text = if (ratioUsed != 1.0 && ratioUsed != lastAutosensData?.autosensResult?.ratio) String.format(Locale.getDefault(), "%.0f%%\n", ratioUsed * 100) else ""
             text += String.format(
-                Locale.getDefault(), "%1$.1f→%2$.1f (%3$.1f)",
+                Locale.getDefault(), "%1$.1f→%2$.1f",
                 profileUtil.fromMgdlToUnits(isfMgdl, profileFunction.getUnits()),
-                profileUtil.fromMgdlToUnits(variableSens, profileFunction.getUnits()),
-                profileUtil.fromMgdlToUnits(isfForCarbs ?: 0.0, profileFunction.getUnits())
+                profileUtil.fromMgdlToUnits(variableSens, profileFunction.getUnits())
             )
             binding.infoLayout.variableSensitivity.text = text
             binding.infoLayout.variableSensitivity.visibility = View.VISIBLE
+            var sensitivityText = rh.gs(app.aaps.core.ui.R.string.isf_for_carbs, profileUtil.fromMgdlToUnits(isfForCarbs ?: 0.0, profileFunction.getUnits()))
+            if (config.APS) {
+                val aps = activePlugin.activeAPS
+                aps.getSensitivityOverviewString()?.let {
+                    sensitivityText += "\n$it"
+                }
+            }
+            runOnUiThread {
+                _binding ?: return@runOnUiThread
+                binding.infoLayout.asLayout.setOnClickListener { activity?.let { OKDialog.show(it, rh.gs(app.aaps.core.ui.R.string.sensitivity), sensitivityText) } }
```

Comment:
this area seems to be too large to me

Reply:
One call of `updateSensitivity()` is from a `runOnUiThread` block and the second call is within a `.subscribe(...)` / rxBus 🤔

All the other "setOnClickListener" (except those within `onViewCreated()` were into a dedicated UiThread. that's why I included the bock to be sure...

If you think I can remove it I'll push a commit to simplify code...